manifest.md

```diff
@@ -23,6 +23,7 @@ A client will distinguish a manifest list from an image manifest based on the Co
 
   This REQUIRED property contains the MIME type of the manifest list.
   For this version of the specification, this MUST be set to `application/vnd.oci.image.manifest.list.v1+json`.
+  For the MIME type that this is compatible with see the [matrix](media-types.md#compatibility-matrix).
```

Comment:
Yeah MIME was in the prior sentence already so I just ran with it.  Will remove MIME type from the doc.